src/Stripe.net/Entities/Customers/Customer.cs

```diff
@@ -164,15 +164,23 @@ internal object InternalDefaultSource
         [JsonProperty("subscriptions")]
         public StripeList<Subscription> Subscriptions { get; set; }
 
+        /// <summary>
+        /// The customer’s current tax ids, if any.
+        /// </summary>
+        [JsonProperty("tax_ids")]
+        public StripeList<TaxId> TaxIds { get; set; }
```

Comment:
Should we also update `CustomerTaxIdOptions`? https://github.com/stripe/stripe-dotnet/tree/master/src/Stripe.net/Services/Customers
https://site-admin.stripe.com/docs/api/customers/create#create_customer-tax_id_data